apps/server/src/modules/task-card/uc/task-card.uc.spec.ts

```diff
@@ -274,35 +294,21 @@ describe('TaskCardUc', () => {
 			expect((result.card.cardElements.getItems()[0] as RichTextCardElement).value).toEqual(richText[0]);
 			expect((result.card.cardElements.getItems()[1] as RichTextCardElement).value).toEqual(richText[1]);
 		});
-		it('should return the task card and task without course if no courseId is given', async () => {
-			const taskCardCreateParamsWithoutCourse = {
-				title,
-				text: [
-					new RichText({ content: richText[0], type: InputFormat.RICH_TEXT_CK5 }),
-					new RichText({ content: richText[1], type: InputFormat.RICH_TEXT_CK5 }),
-				],
-				visibleAtDate,
-				dueDate,
-			};
-			const result = await uc.create(user.id, taskCardCreateParamsWithoutCourse);
-			expect(result.card.task.course).not.toBeDefined();
-			expect(result.taskWithStatusVo.task.course).not.toBeDefined();
-		});
-		it('should return the task card with default visible at date and due date if params are not given and creator does NOT provide current school year', async () => {
+		it('should return the task card with default visible at date if params are not given', async () => {
 			const taskCardCreateDefaultParams = {
 				title,
 				text: [
 					new RichText({ content: richText[0], type: InputFormat.RICH_TEXT_CK5 }),
 					new RichText({ content: richText[1], type: InputFormat.RICH_TEXT_CK5 }),
 				],
+				courseId: course.id,
+				dueDate: tomorrow,
 			};
 			const result = await uc.create(user.id, taskCardCreateDefaultParams);
 			const expectedVisibleAtDate = new Date();
-			const expectedDueDate = new Date(new Date().getFullYear() + 1, 11, 31);
 			expect(result.card.visibleAtDate).toEqual(expectedVisibleAtDate);
-			expect(result.card.dueDate).toEqual(expectedDueDate);
 		});
-		it('should return the task card with default visible at date and due date if params are not given and creator does provide current school year', async () => {
+		it('should return the task card with default visible at date if params are not given', async () => {
```

Comment:
here the scenario `should return the task card with default visible at date if params are not given` is duplicated.
before we needed two separate cases as for the dueDate, but as this is not needed anymore, one of the scenarios is enough